packages/flutter_tools/lib/src/base/logger.dart

```diff
@@ -214,6 +214,8 @@ class AnsiTerminal {
   static const String _reset = '\u001B[0m';
   static const String _clear = '\u001B[2J\u001B[H';
 
+  static const int _eNoTty = 25;
```

Comment:
FWIW I believe the correct capitalisation would be eNotTy (the original message was "Not a Typewriter"), but to avoid the confusion maybe _ENOTTY is the way to go :-P